### PR TITLE
fix: add support for string array attribute type

### PR DESCRIPTION
**Pull request by tim-mwangi**

### CODE REVIEW

File: hypertrace-core-graphql-common-schema/src/test/java/org/hypertrace/core/graphql/common/utils/attributes/AttributeTypeConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import org.hypertrace.core.graphql.attributes.AttributeModelType;
+import org.hypertrace.core.graphql.common.schema.attributes.AttributeType;
+import org.junit.jupiter.api.Test;
+
+public class AttributeTypeConverterTest {
+  @Test
+  void testConvert() {
+    AttributeTypeConverter attributeTypeConverter = new AttributeTypeConverter();
+
+    assertEquals(Single.just(AttributeType.STRING).blockingGet(),
```
Comment (aaron-steinfeld) [inline hypertrace-core-graphql-common-schema/src/test/java/org/hypertrace/core/graphql/common/utils/attributes/AttributeTypeConverterTest.java:15]: nit:
```suggestion
    assertEquals(AttributeType.STRING,

**Review by aaron-steinfeld — DISMISSED** [review]

File: hypertrace-core-graphql-common-schema/src/test/java/org/hypertrace/core/graphql/common/utils/attributes/AttributeTypeConverterTest.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        attributeTypeConverter.convert(AttributeModelType.STRING).blockingGet());
+    assertEquals(Single.just(AttributeType.BOOLEAN).blockingGet(),
```
Comment (tim-mwangi) [inline hypertrace-core-graphql-common-schema/src/test/java/org/hypertrace/core/graphql/common/utils/attributes/AttributeTypeConverterTest.java:17]:
```suggestion
    assertEquals(Single.just(AttributeType.BOOLEAN).blockingGet(),
```
```suggestion
    assertEquals(AttributeType.BOOLEAN,
```

File: hypertrace-core-graphql-common-schema/src/test/java/org/hypertrace/core/graphql/common/utils/attributes/AttributeTypeConverterTest.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        attributeTypeConverter.convert(AttributeModelType.BOOLEAN).blockingGet());
+    assertEquals(Single.just(AttributeType.LONG).blockingGet(),
```
Comment (tim-mwangi) [inline hypertrace-core-graphql-common-schema/src/test/java/org/hypertrace/core/graphql/common/utils/attributes/AttributeTypeConverterTest.java:19]:
```suggestion
    assertEquals(AttributeType.LONG,
```

File: hypertrace-core-graphql-common-schema/src/test/java/org/hypertrace/core/graphql/common/utils/attributes/AttributeTypeConverterTest.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        attributeTypeConverter.convert(AttributeModelType.LONG).blockingGet());
+    assertEquals(Single.just(AttributeType.DOUBLE).blockingGet(),
```
Comment (tim-mwangi) [inline hypertrace-core-graphql-common-schema/src/test/java/org/hypertrace/core/graphql/common/utils/attributes/AttributeTypeConverterTest.java:21]:
```suggestion
    assertEquals(AttributeType.DOUBLE,
```

File: hypertrace-core-graphql-common-schema/src/test/java/org/hypertrace/core/graphql/common/utils/attributes/AttributeTypeConverterTest.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+        attributeTypeConverter.convert(AttributeModelType.DOUBLE).blockingGet());
+    assertEquals(Single.just(AttributeType.TIMESTAMP).blockingGet(),
```
Comment (tim-mwangi) [inline hypertrace-core-graphql-common-schema/src/test/java/org/hypertrace/core/graphql/common/utils/attributes/AttributeTypeConverterTest.java:23]:
```suggestion
    assertEquals(AttributeType.TIMESTAMP,
```

File: hypertrace-core-graphql-common-schema/src/test/java/org/hypertrace/core/graphql/common/utils/attributes/AttributeTypeConverterTest.java [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+        attributeTypeConverter.convert(AttributeModelType.TIMESTAMP).blockingGet());
+    assertEquals(Single.just(AttributeType.STRING_MAP).blockingGet(),
```
Comment (tim-mwangi) [inline hypertrace-core-graphql-common-schema/src/test/java/org/hypertrace/core/graphql/common/utils/attributes/AttributeTypeConverterTest.java:25]:
```suggestion
    assertEquals(AttributeType.STRING_MAP,
```

File: hypertrace-core-graphql-common-schema/src/test/java/org/hypertrace/core/graphql/common/utils/attributes/AttributeTypeConverterTest.java [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+        attributeTypeConverter.convert(AttributeModelType.STRING_MAP).blockingGet());
+    assertEquals(Single.just(AttributeType.STRING_ARRAY).blockingGet(),
```
Comment (tim-mwangi) [inline hypertrace-core-graphql-common-schema/src/test/java/org/hypertrace/core/graphql/common/utils/attributes/AttributeTypeConverterTest.java:27]:
```suggestion
    assertEquals(AttributeType.STRING_ARRAY,
```

**Review by aaron-steinfeld — APPROVED** [review]

**Pull request merged**